docs/body.md

```diff
@@ -459,6 +459,16 @@ This ZenPack still supports the previous configuration, where messages were sent
 
 For new openstack installs, we recommend using the RabbitMQ-Ceilometer endpoint, which is what will be reported by `openstack_amqp_config`.
 
+When the RabbitMQ-Ceilometer service is added to Control Center, two IP Assignments will be added as well.  Upon creation, they will have no IP address, only a port number.  You must add the IP address by following these steps:
```

Comment:
I believe one is for amqp, and the other is for the admin API.  We use the admin API to automatically set user/pass/perm internally.  It probably doesn't need an address assignment, actually, since we don't hit it externally, although they could use it to get to the RabbtiMQ UI for that collector.